bdd100k/common/seg.toml

```diff
@@ -0,0 +1,23 @@
+sem_seg = [
```

Comment:
shall we put them in `config.toml`? Or is that only for coco conversion?

Reply:
I re-check it, find it is not only for coco conversion. So I will move them to config.toml